src/main/java/forceitembattle/commands/player/CommandPosition.java

```diff
@@ -45,89 +45,105 @@ public void onPlayerCommand(Player player, String label, String[] args) {
+            player.sendMessage(this.plugin.getGamemanager().getMiniMessage().deserialize(prefix + "<gray>All locations have been removed."));
             return;
         }
 
         if (!this.plugin.getPositionManager().positionExist(locationName)) {
-            player.sendMessage(prefix + "§cPosition §f" + locationName + " §cdoes not exist.");
+            player.sendMessage(this.plugin.getGamemanager().getMiniMessage().deserialize(prefix + "<red>Position <white>" + locationName + " <red>does not exist."));
             return;
         }
 
         this.plugin.getPositionManager().removePosition(locationName);
-        player.sendMessage(prefix + "§7Position §3" + locationName + " §7has been removed.");
+        player.sendMessage(this.plugin.getGamemanager().getMiniMessage().deserialize(prefix + "<gray>Position <dark_aqua>" + locationName + " <gray>has been removed."));
     }
 
     // Utility stuff
 
     private String locationToString(Location location) {
         if (location.getWorld() == null) {
-            return "§cunknown location";
+            return "<red>unknown location";
         }
 
-        return "§3" + location.getBlockX() + "§7, §3" + location.getBlockY() + "§7, §3" + location.getBlockZ();
+        return "<dark_aqua>" + location.getBlockX() + "<gray>, <dark_aqua>" + location.getBlockY() + "<gray>, <dark_aqua>" + location.getBlockZ();
     }
 
     private String distance(Location playerLocation, Location destination) {
         if (playerLocation.getWorld() == null || destination.getWorld() == null) {
-            return " §c(unknown)";
+            return " <red>(unknown)";
         }
 
         if (!playerLocation.getWorld().equals(destination.getWorld())) {
-            return " §7in the " + getWorldName(destination.getWorld());
+            return " <gray>in the " + getWorldName(destination.getWorld());
         }
 
 
-        return " §a(" + (int) playerLocation.distance(destination) + " blocks away)";
+        return " <green>(" + (int) playerLocation.distance(destination) + " blocks away)";
     }
 
     private String getWorldName(World world) {
         if (world == null) {
-            return "§8unknown";
+            return "<dark_gray>unknown";
         }
 
         String worldName = world.getName();
 
         if (worldName.contains("nether")) {
-            return "§cnether";
+            return "<red>nether";
         }
 
         if (worldName.contains("end")) {
-            return "§eend";
+            return "<yellow>end";
         }
 
-        return "§aoverworld";
+        return "<green>overworld";
     }
+
+    private void playParticleLine(@NonNull Player player, @NonNull Location position) {
+        if (player.getWorld() != position.getWorld()) return;
+
+        // Defining target location to
+        Location target = position.clone().add(0, 0.3, 0);
+
+        final int[] current = {0};
+        Bukkit.getScheduler().runTaskTimer(plugin, task -> {
```

Comment:
Maybe it would be a bit more elegant to create an extra class for these particle lines that has the counter in it as variable.  Or you could use something like this:
```java
new BukkitRunnable() {
    int current = 0;
    
    @Override
    public void run() {
        if (++current == 20) {
            cancel();
        }
        // stuff
    }
}.runTaskTimer(plugin, 0, 10);
```

Reply:
I dont think it is that necessary to create an extra class for only this line. Instead I used your idea